lisp/Makefile

```diff
@@ -37,9 +43,44 @@ all: julia-mode.el $(ELC) ess-custom.el ess-autoloads.el
 .PHONY: dist
 dist: all
 
-install: dist
+dist/package.out: 
+	$(INSTALLDIR) dist
+	if [ ! -s "dist/$(PKBUILD).tar.gz" ] ; then \
+	  cd dist ; curl -sLOk https://github.com/melpa/package-build/archive/$(PKBUILD).tar.gz ; fi
+	cd dist ; tar xfz $(PKBUILD).tar.gz
+	cd dist/package-build-$(PKBUILD) ; make -s loaddefs
+	$(INSTALLDIR) dist/recipes
+	if [ ! -s "dist/recipes/ess" ] ; then \
+	  cd dist/recipes ; curl -sLOk https://raw.githubusercontent.com/melpa/melpa/master/recipes/ess ; fi
+	$(EMACSBATCH) -L dist/package-build-$(PKBUILD) \
+	--eval "(require 'package-build)" \
+	--eval "(setq debug-on-error t)" \
+	--eval "(setq package-user-dir (expand-file-name \"dist\"))" \
+	--eval "(package-initialize)" \
+	--eval "(add-to-list 'package-archives '(\"melpa\" . \"http://melpa.org/packages/\"))" \
+	--eval "(package-refresh-contents)" \
+	--eval "(setq rcp (package-recipe-lookup \"ess\"))" \
+	--eval "(unless (file-exists-p package-build-archive-dir) \
+	           (make-directory package-build-archive-dir))" \
+	--eval "(unless (zerop (length \"$(TRAVIS_PULL_REQUEST_SLUG)\")) \
+	          (oset rcp :repo \"$(TRAVIS_PULL_REQUEST_SLUG)\") \
```

Comment:
Can we avoid these variables? Would be nice to be able to run it locally.

Reply:
it can be run locally.  perhaps you should try (although you would need a Makeconf.local)